ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -385,14 +384,6 @@ def write_ml_model(self, model: CreateMlModelEntityRequest):
             logger.error(err)
             self.status.failure(f"Model: {model.name}")
 
-    def _bootstrap_entities(self):
-        team_response = self.api_client.get("/teams")
-        for team in team_response["data"]:
-            self.team_entities[team["name"]] = team["id"]
```

Comment:
Are these variables self.team_entities and self.role_entities used elsewhere in this file?

Reply:
Yes, these are used in _create_team and _create_role methods respectively.

Reply:
Also the write_users method which I've modified to include the logic of _bootstrap_entities uses these variables.